src/ContMechTensors.jl

```diff
@@ -315,7 +352,7 @@ for TensorType in (SymmetricTensor, Tensor)
             exp = tensor_create(get_type(Tt),f)
             return quote
                 $(Expr(:meta, :inline))
-                $($TensorType){order, dim, T, $N}($exp)
```

Comment:
Yea I am cleaning up. Also all the `Tv` we calculate can be removed with this method, which I like